src/app/graphics/svg-generator.ts

```diff
@@ -20,17 +21,39 @@ function renderDefs() {
     <style>
       .rectangle {
         fill: transparent;
-        stroke: black;
-        stroke-width: 3;
+        stroke: #8bc34a;
+        stroke-width: 4;
+      }
+
+      .rectangle.errorOutline {
+        stroke: #ec4758;
       }
```

Comment:
![image](https://github.com/frankframework/frank-config-layout/assets/6784732/be20ad90-d4bb-4198-a18e-d21e907ae9b4)

Als je de Karma test fixt zoals ik hier vraag, dan maak je een conflict met mijn PR https://github.com/frankframework/frank-config-layout/pull/6. Mag mijn PR voorgaan?

Reply:
Niet meer van toepassing. We hebben genoemde PR gesloten.